### ISSUE TITLE
[CLI] Running `rancher server delete` or `rancher server switch` with no Rancher server results in faulty message to `Select a Server`

### ISSUE THREAD

**Issue by markusewalker** [rancher/rancher#37545]
**Rancher Server Setup**
- Rancher version: Rancher CLI `v2.6.5-rc1`
- Installation option (Docker install/Helm Chart): N/A
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):
- Proxy/Cert Details: N/A
   
**Describe the bug**

When working with the Rancher CLI, if there are currently no servers in the `cli2.json`, the `rancher server switch` and `rancher server delete` commands do the following:

```
root@localhost:~# rancher server ls
CURRENT   NAME      URL
# rancher server switch
INDEX     NAME      URL
Select a Server:1
Invalid input, enter a number between 1 and 0: 0
Invalid input, enter a number between 1 and 0: 
^C
root@localhost:~# rancher server delete
INDEX     NAME      URL
Select a Server:1
Invalid input, enter a number between 1 and 0: 0
Invalid input, enter a number between 1 and 0: a
Invalid input, enter a number between 1 and 0: ^C
root@localhost:~#
```
There isn't any server present as shown in `rancher server ls`, so there shouldn't be an option to `Select a Server`. This is a bug and could cause user issues with automation should something happen to their `cli2.json`.

**To Reproduce**
 
1. Download the latest Rancher CLI.
2. Run `rancher server switch` or `rancher server delete` without connecting any Rancher servers.

**Result**
You are prompted to enter in a server despite none being available.

**Expected Result**

I would expect for the `rancher` CLI tool to inform me that there are no servers available. A message such as `No servers are available. Refer to the rancher login command to add your Rancher server.`

**Comment by sowmyav27** [comment]
Validate using Rancher CLI `v2.6.6-rc4`

**Comment by markusewalker** [comment]
Verified that this has been addressed in `v2.6.6-rc4`.

**ENVIRONMENT DETAILS**
- Rancher CLI version: `v2.6.6-rc4`

**TEST RESULT**
PASS

**VERIFICATION STEPS**
1. On a client machine, installed the latest Rancher CLI version `v2.6.6-rc4`.
2. Ran following series of commands with no servers configured; received following error message:
```
# rancher server ls
FATA[0000] no servers are currently configured          
root@localhost:~# rancher server switch
FATA[0000] no servers are currently configured          
root@localhost:~# rancher server delete
FATA[0000] no servers are currently configured  
```
